src/styles/blocks/header.scss

```diff
@@ -0,0 +1,57 @@
+.header {
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  background: #fff;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  display: flex;
+}
+
+.nav {
+  &__list {
+    list-style: none;
+    font-size: 12px;
+    line-height: 15px;
+    text-transform: uppercase;
+    list-style-type: none;
+    justify-content: end;
+    display: flex;
+  }
+
+  &__item:not(:last-child) {
+    margin-right: 20px;
+  }
+
+  &__link {
+    height: 60px;
+    display: block;
+    padding: 23px 0;
+    text-decoration: none;
+    color: $main-color;
+
+    &:hover {
+      color: $button-color;
+    }
+  }
+}
+
+.logo {
+  width: 40px;
+  height: 40px;
+}
+
+.is-active {
+  color: $button-color;
+  position: relative;
+
+  &::after {
+    content: "";
+    position: absolute;
+    bottom: 0;
+    left: 0;
+    right: 0;
+    height: 4px;
+    background-color: $button-color;
+    border-radius: 8px;
+  }
```

Comment:
move this to nav__link as modificator
```
&__link {
 &--active {
  styles
 }
}
```

Reply:
it's special modificator for mentor. It's from previous task.